src/OSWindow-Core/OSWindow.class.st

```diff
@@ -433,7 +454,7 @@ OSWindow >> stopTextInput [
 
 { #category : #accessing }
 OSWindow >> title [
-	^ self validHandle title
+	^ initialAttributes title: (self validHandle title)
```

Comment:
So at the end the initialAttributes do not hold only the initial attributes. They hold the current attributes of the window, right? Could we rename the variable?